packages/core/src/components/Shadows/EditShadowModal.tsx

```diff
@@ -78,6 +79,54 @@ export function EditShadowModal({
     onClose()
   }
 
+  const [color, setColor] = useState(presetColor)
+  const [inputColor, setInputColor] = useState(presetColor)
+  const [hOffset, sethOffset] = useState(initialValues?.hOffset ?? 0)
+  const [vOffset, setVOffset] = useState(initialValues?.vOffset ?? 0)
+  const [blur, setBlur] = useState(initialValues?.blur ?? 0)
+  const [spread, setSpread] = useState(initialValues?.spread ?? 0)
+
+  const codeResult = useMemo(() => {
+    return `${hOffset}px ${vOffset}px ${blur}px ${spread}px ${color}`
+  }, [hOffset, vOffset, blur, spread, color])
```

Comment:
```suggestion
  const codeResult = `${hOffset}px ${vOffset}px ${blur}px ${spread}px ${color}`
```

nit: I think this is an instance where `useMemo`  might actually be less performant because of the extra memory / isn't actually saving too much computation